nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -564,11 +564,12 @@ in
     })
 
     (mkIf serviceCfg.games.enable {
-      environment.systemPackages = (with pkgs.gnome; removePackagesByName [
+      environment.systemPackages = (with pkgs.gnome; with pkgs; removePackagesByName [
         aisleriot
         atomix
         five-or-more
         four-in-a-row
+        gnome-2048
```

Comment:
It would be nicer to  use explicit `pkgs.gnome-2048` to make the origin clearer, rather than crossing `with` streams.